### PR TITLE
UserId selection and notification mods

### PR DESCRIPTION
**Pull request by mwanderson**
Added in the following:
 - Changed the userid drop down to have an onchange event associated with it.  When triggered it will call the existing function that is called when user clicks on the enter button.
 - Added in toastr (https://codeseven.github.io/toastr/).  When the change event is triggered a toastr notification will be fired.
 - Toastr requires a .js and .css file to be added.

### CODE REVIEW

File: content/jr-sa.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        </div>
+      </div>
+    </div>
+
+    <div class="col-sm-3 text-center">
+      <div class="card">
+        <div class="card-body">
+          <a href="https://www.forbes.com/companies/red-hat/#1f9ef992eb7a">
+              <div class="media media-element-container media-default"><img class="media-element file-default ux-image-embed-img" alt="Forbes Most Innovated
+              Companies" title="Forbes Most Innovated Companies" width="225" src="https://www.redhat.com/cms/managed-files/Forbes_Worlds_Most_Innovative_Companies_Icon_color_350x100.png" height="90" /></div></a>
```
Comment (Tekwriter1221) [inline content/jr-sa.html]: Innovative

File: content/workshops/rhel_8/_index.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+:icons: font
+:iconsdir: http://people.redhat.com/~jduncan/images/icons
+:imagesdir: /workshops/rhel_8/images
+
+= Red Hat Enterprise Linux 8 Workshop
+
+== Workshop Description
+
+This workshop will cover the features, specifications and improvements that comprise the release of version 8 of Red Hat Enterprise Linux (RHEL).  RHEL 8 brings many improvements over previous releases, including:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/_index.adoc]: Line 22 “(RHEL 8)” Also, anywhere you use “Version 8” capitalized, as in Red Hat Enterprise Linux Version 8.

File: content/workshops/rhel_8/_index.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+* Application streams, which allow for the use of bleeding edge interpreters, compilers and libraries, while maintaining full backwards compatibility and stability of the system
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/_index.adoc]: Line 24 Replace “allow for” with “enables”

File: content/workshops/rhel_8/_index.adoc [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* The web console has been completely overhauled, and significant new features have been added
+** Session recording capability has been added to the web console
+* Image builder provides for the easy creation of customized RHEL installation images
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/_index.adoc]: Line 27 delete “for”

File: content/workshops/rhel_8/_index.adoc [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* The system package management tool, YUM, has been enhanced with DNF technology, providing significant performance improvements
+* System purpose has been enhanced to automatically tune system performance for the usage of the system
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/_index.adoc]: Line 29 suggest: System features have been enhanced with automatic performance tuning for a better user experience.

File: content/workshops/rhel_8/_index.adoc [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+* There is a new `update-crypto-policies` tool, which applies cryptograhic polices system-wide
+** TLS, IPsec, DNSSEC, Kerberos, and SSH protocols are covered
+* The system Python interpreter is now version 3.6
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/_index.adoc]: Line 32 The system Python interpreter has been updated to version 3.6

File: _index.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+title: Agile Integrations Workshop - Citizen Integrator
+menu:
+  main:
+    parent: workshops
+    pre: fa fa-play-circle
+---
+## Description
+
+The Agile Integration Workshop teaches the development, testing, and deployment of cloud-native and connected solutions.  You will walk through an API Development and Agile Integration process leveraging OpenShift, Apicurio, Microcks, Fuse, and 3scale for a sample application.
```
Comment (Tekwriter1221) [inline _index.md]: Line 10:(Second sentence) In this workshop, you will walk through API Development and an Agile Integration processes that leverage OpenShift, Apicurio.....

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+
+
+= Exercise 1.0 - Welcome to RHEL!
+
+
+== Exercise Description
+
+Welcom to RHEL!
+
+Red Hat Enterprise Linux 8 is the intelligent enterprise Linux, providing a secure and consistent foundation across hybrid cloud deployments and the tools enterprises need to deliver services and workloads faster with less effort.  RHEL 8 enables delivery of any application, on any footprint, at any time.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 20 - consider replacing “footprint” with architecture or framework or something more technical.

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+
+With predictable release cadence and 10-year enterprise support, RHEL 8 is the safe and stable choice for your enterprise.
+
+RHEL 8 also automates regulatory compliance and security configuration remediation ​across your system and within containers​​ with OpenSCAP, Red Hat’s National Institute of Standards and Technology (NIST)-certified scanner.  RHEL 8's security portfolio also includes a new, system-wide encryption policy and the latest nftables/firewalld firewall.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 24: Consider - RHEL 8 uses OpenSCAP on your system, to automate regulatory compliance and security configuration. OpenSCAP is the National Institute of Standards and Technology (NIST)-certified scanner, used by Red Hat.

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+
+To get started, we will take a tour of the web console. The web console (called `cockpit`) will be your primary interactive interface to RHEL. The console has a number of functions that allow you to manage most aspects of the installation.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 26 Replace “allow” with “enable”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+The various functional sections can be seen in `Figure 1`, and are: +
+{empty} +
+*System* +
+Contains an overview of the system, including metadata and health graphs
+{empty} +
+*Logs* +
+Displays current `journald` logs from all services
+{empty} +
+*Storage* +
+Displays storage activity graphs, and logs. Allows for the management of NFS mounts, RAID devices, volume groups, VDO (deduplication) devices, and iSCSI targets
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 44 Replace “Allows for” with “Enables”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 12]
```diff
@@ -0,0 +1,236 @@
...
+Displays network activity, logs, and allows firewall and network maintenance
+{empty} +
+*Podman Containers* +
+Allows for the management of container images and running containers
+{empty} +
+*Virtual Machines* +
+Virtual machines can be created, deleted and controlled
+{empty} +
+*Accounts* +
+Allows for the creation, deletion and configuration of user accounts and ssh keys
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 56 Replace “Allows for” with “Enables”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 13]
```diff
@@ -0,0 +1,236 @@
...
+{empty} +
+*Services* +
+Displays all system targets, services, sockets, timers and paths, as managed by `systemd`, the root process
+{empty} +
+*Session Recording* +
+Manages recorded sessions, and allows for the configuration of session recorder defaults
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 62 Replace “Allows for” with “Enables”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 14]
```diff
@@ -0,0 +1,236 @@
...
+{empty} +
+*Applications* +
+Allows for the installation and removal of web console modules
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 65 Replace “Allows for” with “Enables”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 15]
```diff
@@ -0,0 +1,236 @@
...
+{empty} +
+*Diagnostic Reports* +
+Creates diagnostic reports used by Red Hat support, also known as SOSreports
+{empty} +
+*Kernel Dump* +
+Manages kernel debugging dumps, created when the system crashes
+{empty} +
+*SELinux* +
+Allows for the setting of SELinux to permissive mode, and shows recent SELinux alerts (`AVCs`)
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 74: Enables SELinux to be set to Passive mode and displays recent SELinux alerts (AVCs)

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 16]
```diff
@@ -0,0 +1,236 @@
...
+{empty} +
+*Software Updates* +
+Manages automatic package updating, and allows for the manual installation of updates
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 77 Manages automatic package updates and enables manual update installation.

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 17]
```diff
@@ -0,0 +1,236 @@
...
+Provides an interactive shell
+--
+
+Next, let's proceed to a more detailed examination of each module. Please click through the web console on your host, as you read the explanations, here.
+
+=== Section 2: System Module
+
+image::cockpit_system.png[caption="Figure 2: ", title='Web Console System Module', 500]
+
+The system module is the main screen that you see, when you first login to the web console. It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 92: The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 18]
```diff
@@ -0,0 +1,236 @@
...
+
+* hardware type
+** detailed system hardware data is available, from the link on the main page
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 95 Remove comma

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 19]
```diff
@@ -0,0 +1,236 @@
...
+* asset tag
+* machine ID
+* operating system
+** when updates to the system are available, an "updates available" link will be present
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 99: Replace “will be present” with “displays”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 20]
```diff
@@ -0,0 +1,236 @@
...
+* SSH keys
+** key fingerprints for all created 'root' keys
+* host name
+** the system's name may be changed, and an alias may be added
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 103 Remove comma

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 21]
```diff
@@ -0,0 +1,236 @@
...
+
+Next, let's proceed to a more detailed examination of each module. Please click through the web console on your host, as you read the explanations, here.
+
+=== Section 2: System Module
+
+image::cockpit_system.png[caption="Figure 2: ", title='Web Console System Module', 500]
+
+The system module is the main screen that you see, when you first login to the web console. It contains an overview of the system, including the below sections:
+
+* hardware type
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Lines 94 - 108 Consider capitalizing the field names. (Hardware Type, Asset Tag, etc.)

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 22]
```diff
@@ -0,0 +1,236 @@
...
+* host name
+** the system's name may be changed, and an alias may be added
+* domain
+** the system may be added to an LDAP/AD domain
+* time and date
+** time zone and automatic or manual time and date setting can be done
+* performance profile
+** the system's performance profile may be changed
+
+Additionally, there are graphs that show the system's CPU, memory, disk, and network utilization.  Each graph may be enlarged, by clicking its associated link.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 111 Remove comma

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 23]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 3: Logs Module
+
+image::cockpit_01_logs.png[caption="Figure 3: ", title='Web Console Logs Module', 500]
+
+The logs module provides date-sorted view of recent system log entries.  You can change the date or apply basic filters using the pull-down menus at the top of the page.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 117 The Logs module presents a ….

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 24]
```diff
@@ -0,0 +1,236 @@
...
+
+image::cockpit_02_storage.png[caption="Figure 4: ", title='Web Console Storage Module', 500]
+
+The storage module provides graphs of current storage performance.  It also lists local and network-mounted filesystems, plus recent storage-related log entries.
+
+=== Section 5: Networking Module
+
+image::cockpit_03_networking.png[caption="Figure 5: ", title='Web Console Networking Module', 500]
+
+The network module contains performance graphs similar to the storage module.  Additionally, it shows the configuration of your network interfaces, identifies unused interfaces, and provides access to, and control of, the system firewall.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 129 Replace “shows” with “displays”

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 25]
```diff
@@ -0,0 +1,236 @@
...
+
+image::cockpit_04_podman.png[caption="Figure 6: ", title='Web Console Podman Containers Module', 500]
+
+This module provides control of local podman containers and their corresponding images.
+
+=== Section 6: Virtual Machines Module
+
+image::cockpit_05_vm.png[caption="Figure 7: ", title='Web Console Virtual Machines Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 141 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 26]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 7: Accounts Module
+
+image::cockpit_06_accounts.png[caption="Figure 8: ", title='Web Console Accounts Module', 500]
+
+The accounts module allows you to create and modify local user accounts.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 147 Replace “allows” with “enables’

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 27]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 8: Services Module
+
+image::cockpit_07_services.png[caption="Figure 9: ", title='Web Console Services Module', 500]
+
+The services module allows basic administration of systemd objects, including system services.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 153 Replace “allows” with “enables’

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 28]
```diff
@@ -0,0 +1,236 @@
...
+
+image::cockpit_08_sr.png[caption="Figure 10: ", title='Web Console Session Recording Module', 500]
+
+This module provides management of recorded terminal sessions. 
+
+=== Section 10: Applications Module
+
+image::cockpit_09_applications.png[caption="Figure 11: ", title='Web Console Applications Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 165 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 29]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 11: Diagnostic Reports Module
+
+image::cockpit_10_dr.png[caption="Figure 12: ", title='Web Console Diagnostic Reports Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 171 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 30]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 12: Kernel Dump Module
+
+image::cockpit_11_kd.png[caption="Figure 13: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 177 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 31]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 13: SELinux Module
+
+image::cockpit_12_selinux.png[caption="Figure 14: ", title='Web Console SELinux Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 183 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 32]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 14: Software Update Module
+
+image::cockpit_13_su.png[caption="Figure 15: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 189 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 33]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 15: Subscriptions Module
+
+image::cockpit_14_subscriptions.png[caption="Figure 16: ", title='Web Console Subscriptions Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 195 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 34]
```diff
@@ -0,0 +1,236 @@
...
+
+=== Section 16: Terminal Module
+
+image::cockpit_15_terminal.png[caption="Figure 17: ", title='Web Console Terminal Module', 500]
+
+The terminal module provides command-line access to the system, similar to an SSH session.  Section 16.1 has some example commands to try if you'd like to test it out.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 201 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics:

File: content/workshops/rhel_8/exercise1.0.adoc [hunk 35]
```diff
@@ -0,0 +1,236 @@
...
+nft is the nftables utility.  Similar to ''iptables -L'', the following lists the system's active firewall ruleset:
+
+[source, bash]
+----
+nft list ruleset
+----
+
+=== Conclusion
+
+The web console has a lot of new functionality, in version 8 of Red Hat Enterprise Linux. Hopefully, this brief (!) overview has demonstrated that, and given you a taste of how easily systems may be managed, with this new tool.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.0.adoc]: Line 234 The web console in Red Hat Enterprise Linux Version 8 has a lot of new functionality. Hopefully, this brief (!) overview has provided a comprehensive description of the web console features and provided you with a taste of how you can easily manage your systems with this new tool.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 36]
```diff
@@ -0,0 +1,137 @@
...
+:package_url: http://docs.ansible.com/ansible/package_module.html
+:service_url: http://docs.ansible.com/ansible/service_module.html
+:yaml_url: http://docs.ansible.com/ansible/YAMLSyntax.html
+
+
+= Exercise 1.1 - Managing Updates Using the Web Console
+
+== Exercise Description
+
+Now that you've toured the web console, lets work through updating the system.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 22 let’s

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 37]
```diff
@@ -0,0 +1,137 @@
...
+
+= Exercise 1.1 - Managing Updates Using the Web Console
+
+== Exercise Description
+
+Now that you've toured the web console, lets work through updating the system. 
+
+== Section 1: Basic Updating
+
+As you did in the previous section, navigate to the `Software Updates` section of the Web Console, by clicking on the `Software Updates` link, in the left-hand menu. There will then be a brief interval where `Refreshing package information` will appear, along with a progress bar. Once that completes, you can proceed with the exercise.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 26: 1) “Web Console” has been lower case to this point. 2) Change “by clicking on the” to “by clicking the”.  3) Remove the comma after ‘Software Updates’ link. 4) A ‘Refreshing package information’ message will briefly display with a progress bar. Upon completions of the refresh, you may proceed with the exercise.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 38]
```diff
@@ -0,0 +1,137 @@
...
+
+=== Step 1: Check for available updates
+
+Take a look at the `Available Updates` section of the browser window.  You should see several packages listed, in a single group, like this:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 30 Replace “should” with “will”

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 39]
```diff
@@ -0,0 +1,137 @@
...
+
+image::su_updates.png[Software Updates, 900, caption="Figure 1: ", title="Packages Available for Updating"]
+
+You can expand the list, to see more details, by clicking on the drill-down `>` link:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 34: Click the drill-down to expand the list and view details

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 40]
```diff
@@ -0,0 +1,137 @@
...
+
+image::su_updates_expanded.png[Software Updates, 900, caption="Figure 2: ", title="Expanded Updates"]
+
+In the expanded view, you can see details on what CVEs are relevant to the updates, and any associated bug IDs.  Additionally, a summary of the changes is displayed.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 38 Replace “see” with “view”

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 41]
```diff
@@ -0,0 +1,137 @@
...
+
+=== Step 2: Install updates
+
+Once you are finished reading, click the `Install Security Updates` button, to install the updated packages.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 42 After viewing the available updates, click ‘Install Security Updates’ to install the updated packages

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 42]
```diff
@@ -0,0 +1,137 @@
...
+
+The updates will install, and while doing so, there will be information on what actions are taking place, and below that, an expandable log drill-down. Once complete, you may receive a window like this:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 44 As the updates are installing, the system will display the progress of the actions taking place and an expandable log drill-down.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 43]
```diff
@@ -0,0 +1,137 @@
...
+
+image::su_updates_restart.png[Software Updates, 900, caption="Figure 3: ", title="Updates Recommend a Restart"]
+
+=== Step 2: Restart the system
+
+Click the `Restart Now` button, to reboot your managed host, and cause the changes to take effect. You will see screens like these, while the system is rebooting:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 50 Click the `Restart Now` button, to reboot your managed host and initiate the updates. Screens similar to those below will display while the system is rebooting:

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 44]
```diff
@@ -0,0 +1,137 @@
...
+
+image::su_updates_restarting.png[Software Updates, 900, caption="Figure 4: ", title="System Restarting"]
+
+image::su_updates_disconnected.png[Software Updates, 900, caption="Figure 5: ", title="Disconnected"]
+
+=== Step 3: Reconnect
+
+Wait for a couple of minutes, for the system to come back up, and then click the `Reconnect` button, to be returned to the login screen, for the web console. Login, as before.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 58 Once the system has restarted, click the Reconnect button to return to the web console Log In screen. Then log in, as usual.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 45]
```diff
@@ -0,0 +1,137 @@
...
+
+=== Step 4: Summary
+
+Once you are logged back in, you will again see the updates screen, with a list of the updates that you just applied. You can expand the history list, to see the details.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 62 Once you are logged back in, you will again see the updates screen, with a list of the updates that you just applied. You can expand the history list, to see the details.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 46]
```diff
@@ -0,0 +1,137 @@
...
+
+image::su_updates_summary.png[Software Updates, 900, caption="Figure 6: ", title="Update Summary"]
+
+== Section 2: Automatic Updating
+
+Next, let's take a look at automatic updating.  There is an `Automatic Updates` switch, at the top of the updates window.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 68 Next, we will discuss automatic updating. In the web console, you can choose to apply all updates, or security updates and also manage periodicity and time of your automatic updates.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 47]
```diff
@@ -0,0 +1,137 @@
...
+
+=== Step 1: Enable Automatic Updates
+
+Click on the switch, next to the `Automatic Updates` label. You will then see a message, telling you that the package `dnf-automatic` is needed:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 72 Move the Automatic Updates switch to “on.” A message will display indicating a requirement for dnf-automatic.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 48]
```diff
@@ -0,0 +1,137 @@
...
+* 17:00
+* 18:00
+* 19:00
+* 20:00
+* 21:00
+* 22:00
+* 23:00
+|===
+
+Set the automatic update mechanism for Fridays, to apply security updates, at 03:00.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 129 Set the automatic update feature to apply security updates every Friday at 03:00.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 49]
```diff
@@ -0,0 +1,137 @@
...
+
+There isn't much else to updating, via the web console, on RHEL 8.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: Line 131 Remove There isn't much else to updating, via the web console, on RHEL 8, as it provides no value.

File: content/workshops/rhel_8/exercise1.1.adoc [hunk 50]
```diff
@@ -0,0 +1,137 @@
...
+
+== Summary
+
+We've tried out the manual and automatic updating functionality available, in the web console, for RHEL 8.  Please ask any questions that seem relevant, and then we can move on to the next section.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.1.adoc]: 135 We’ve tried out the RHEL 8 web console’s manual and automatic updating functionality. Please ask……

File: content/workshops/rhel_8/exercise1.2.adoc [hunk 51]
```diff
@@ -0,0 +1,158 @@
...
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+:package_url: http://docs.ansible.com/ansible/package_module.html
+:service_url: http://docs.ansible.com/ansible/service_module.html
+:gather_facts_url: http://docs.ansible.com/ansible/latest/playbooks_variables.html#turning-off-facts
+
+
+= Exercise 1.2 - Managing Software in an Application Stream
+
+Application streams are a new capability, in RHEL 8, providing access to the latest tools, languages and libraries, without affecting core system components. This is done by including all of those `streams` in containers. The containers are automatically instantiated, when needed, and go away, when they are not in use.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise1.2.adoc]: Line 17 Application streams are a new RHEL 8 capability that provides access to the latest tools, languages and libraries, without affecting core system components. ……

File: content/workshops/rhel_8/exercise2.0.adoc [hunk 52]
```diff
@@ -0,0 +1,126 @@
...
+
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+
+
+= Exercise 2.0 - Installing Ansible Tower
+
+== Exercise Description
+
+In this exercise, we are going to get Ansible Tower installed on your control node.
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise2.0.adoc]: Line 18 (add) Red Hat Ansible helps teams manage complex, multitier deployments by adding control, knowledge, and delegation to Ansible-powered environments. Red Hat Ansible Tower helps you scale IT automation, manage complex deployments, and speed productivity. In this exercise, we will install Ansible Tower on your control node.

File: content/workshops/rhel_8/exercise2.0.adoc [hunk 53]
```diff
@@ -0,0 +1,126 @@
...
+----
+
+[NOTE]
+Step 7 will take approx. 10-15 minutes to complete.  This may be a good time to take a break.
+
+
+=== Step 8: Confirm results
+
+At this point, your Ansible Tower installation should be complete.
+You can access your Tower workshop (not forgetting that *workshopname* is the name of your workshop, and *#* is your student number) at:
```
Comment (Tekwriter1221) [inline content/workshops/rhel_8/exercise2.0.adoc]: Line 112: Replace “not forgetting that” with “Remember:”

File: content/workshops/rhel_8/_index.adoc (hunk 2 above)
Comment (ajacocks) [reply content/workshops/rhel_8/_index.adoc]: Fixed

File: content/jr-sa.html (hunk 1 above)
Comment (ajacocks) [reply content/jr-sa.html]: Fixed

File: content/workshops/rhel_8/_index.adoc (hunk 3 above)
Comment (ajacocks) [reply content/workshops/rhel_8/_index.adoc]: Fixed

File: content/workshops/rhel_8/_index.adoc (hunk 4 above)
Comment (ajacocks) [reply content/workshops/rhel_8/_index.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.0.adoc (hunk 8 above)
[the remark this answers is not in the document]
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.0.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.0.adoc (hunk 9 above)
[the remark this answers is not in the document]
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.0.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.0.adoc (hunk 10 above)
[the remark this answers is not in the document]
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.0.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.0.adoc (hunk 11 above)
[the remark this answers is not in the document]
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.0.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.0.adoc (hunk 12 above)
[the remark this answers is not in the document]
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.0.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.1.adoc (hunk 49 above)
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.1.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.1.adoc (hunk 50 above)
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.1.adoc]: Fixed

File: content/workshops/rhel_8/exercise1.2.adoc (hunk 51 above)
Comment (ajacocks) [reply content/workshops/rhel_8/exercise1.2.adoc]: Fixed

File: content/workshops/rhel_8/exercise2.0.adoc (hunk 53 above)
Comment (ajacocks) [reply content/workshops/rhel_8/exercise2.0.adoc]: This section has been removed

File: content/workshops/rhel_8/exercise2.0.adoc (hunk 52 above)
Comment (ajacocks) [reply content/workshops/rhel_8/exercise2.0.adoc]: Removed this section

**Pull request merged**